packages/desktop-client/src/components/manager/ConfigServer.js

```diff
@@ -1,25 +1,41 @@
-import React, { useState, useEffect, useCallback } from 'react';
+import React, { useState, useEffect } from 'react';
 import { useDispatch } from 'react-redux';
 import { useHistory } from 'react-router-dom';
-import * as actions from 'loot-core/src/client/actions';
 import {
   View,
   Text,
   Button,
-  ButtonWithLoading
+  ButtonWithLoading,
+  ExternalLink
 } from 'loot-design/src/components/common';
-import { colors, styles } from 'loot-design/src/style';
+import { colors } from 'loot-design/src/style';
 import { signOut, loggedIn } from 'loot-core/src/client/actions/user';
 import { send } from 'loot-core/src/platform/client/fetch';
-import {
-  useBootstrapped,
-  Title,
-  Input,
-  Link,
-  ExternalLink
-} from './subscribe/common';
+import { Title, Input } from './subscribe/common';
+import { useSetThemeColor } from 'loot-design/src/components/hooks';
+import useServerVersion from '../../hooks/useServerVersion';
+
+function Version() {
+  const version = useServerVersion();
+
+  return (
+    <ExternalLink
+      style={{
+        color: colors.n7,
+        margin: 15,
+        marginRight: 17,
+        ':hover': { color: colors.n2 },
+        zIndex: 5001
+      }}
+      href={'https://actualbudget.com/blog/' + window.Actual.ACTUAL_VERSION}
+    >
+      App: v${window.Actual.ACTUAL_VERSION} | Server: ${version}
```

Comment:
Doesnt need the $s as it's no longer a template string 
```js
      App: v{window.Actual.ACTUAL_VERSION} | Server: {version}
```

Reply:
Thanks. Lemme pull in your changes here better. Looks like I was hurrying too fast to refresh my branch.